packages/ui/src/ui/containers/ACL/AclModeControl.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {RadioButton} from '@gravity-ui/uikit';
+
+import {AclMode} from '../../constants/acl';
+import format from '../../common/hammer/format';
+import {ACLReduxProps} from './ACL-connect-helpers';
+
+export function AclModeControl({
+    updadeAclFilters,
+    aclMode,
+}: Pick<ACLReduxProps, 'aclMode' | 'updadeAclFilters'>) {
+    const options = React.useMemo(() => {
```

Comment:
An array of 2 elements is not computationally heavy. You don't have to use useMemo.

This construct does not use props variables. It can be moved from the component to a constant